platform/web-security/src/main/resources/webapp/messages/messages.properties

```diff
@@ -115,6 +115,7 @@ security.confirmationRequired=Password confirmation is required
 security.emailTaken=Email already in use
 security.confirm=Confirm
 security.confirm.permissionDelete=Delete this permission?
+security.defaultPermission=You can not delete a default permission
```

Comment:
maybe change this to 'predefined' permission - default doesn't really fit that well here